index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <title>Behnam Aghaali</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+        <link rel="stylesheet" href="index.css"/>
+    </head>
+    <body>
+        <main>
+            <nav>
+                <img src="images/logo.png" alt="logo"/>
+                <i class="fa fa-bars"></i>
+            </nav>
+            <header>
+                <h1>I'm <b>Behnam</b>.<span> Welcome to my website!</span> </h1>
+                <p>I am a professional web developer. I have been working as a software developer since 2016. 
+                You can find a couple of my works here. If you like what you see here feel free to contact me. 
+                I will be happy if I can help you with your software needs. 
+                </p>
+                <span >Follow me on social media</span>
+                <a href="https://twitter.com/behnamagh1369" class="social"><i class="fa fa-2x fa-twitter"></i></a>
+                <a href="https://www.linkedin.com/in/behnam-aghaali-62561375" class="social"><i class="fa fa-2x fa-linkedin"></i></a>
+                <a href="https://github.com/Behnam1369" class="social"><i class="fa fa-2x fa-github"></i></a>
```

Comment:
- [ ] In the requirements of this project I believe that you are asked to uses a `ul` tag to group the list of icons , and `li` per icon; each icon has a link (<a> tag), that will make your code more semantic and readable. KIndly, work on refactoring that 😸

Reply:
Hello Hamza 
Thank you for your comments. Sure your comments are true and I am going to fix them. 
I just want to know where are the requirements list? I didn't see any requirement about putting the logo inside an `a` tag or using `ul` and `li` tags for social icons. 

I am seeing the project requirements here: 
[](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

Am I in the right place? Or I need to check somewhere else to see the requirements?